tests/ha/qnetd.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test qdevice/qnetd
+# qdevice/qnetd is a supported feature since 15-SP1
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+use version_utils 'is_sle';
+use utils qw(systemctl file_content_replace zypper_call);
+
+sub qdevice_status {
+    my ($expected_status) = @_;
+    my $quorum_status_cmd = 'crm corosync status quorum';
+    my $qnetd_status_cmd  = 'crm corosync status qnetd';
+    my $output            = '';
+
+    # Check qdevice status
+    script_run "$qnetd_status_cmd";
+    #save_screenshot;
+    assert_script_run "$quorum_status_cmd";
+    #save_screenshot;
```

Comment:
No comment code in PR please :-)

Reply:
oups! Thanks for catching this! I forgot to remove my debug mode :)